.github/workflows/api-gateway-ci.yml

```
@@ -0,0 +1,136 @@
+name: API Gateway CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)